contribute/design/openebs-provisioner-high-availibility.md

```diff
@@ -0,0 +1,98 @@
+# High Availability for OpenEBS Volume Provisioner
+---
+## Problems addressed:
+
+- `openebs-provisioner` should always be available.
+- `openebs-provisioner` should always be running on the specified/labeled pods. 
+- If any node in cluster goes down in that scenario the pod should be assigned to other node in the cluster.
+---
+## Solutions for above problems: 
+
+
+- Increase the `replicas` to 2 or >2 in `openebs-operator.yaml` for `openebs-provisioner` deployment. 
+
+
+- We need to make sure that each node is running at least one  `openebs-provisioner` pod.  
```

Comment:
what is there are 3 nodes or 10 nodes, will the provisioner run on each node of the 10 nodes?